pkgs/build-support/fetchgithub/default.nix

```diff
@@ -1,7 +1,7 @@
 { lib, fetchgit, fetchzip }:
 
 { owner, repo, rev, name ? "source"
-, fetchSubmodules ? false, leaveDotGit ? null
+, fetchSubmodules ? false, leaveDotGit ? false
```

Comment:
Looking at `fetchgit`, I think this is needed to allow `leaveDotGit` to be implied by `deepClone`. See also line 51, where we filter it out when `null`.

Reply:
hmmm, right.